packages/amazonq/test/unit/codewhisperer/util/crossFileContextUtil.test.ts

```diff
@@ -37,11 +37,16 @@ describe('crossFileContextUtil', function () {
 
     describe('fetchSupplementalContextForSrc', function () {
         beforeEach(async function () {
+            sinon.restore()
```

Comment:
is it necessary to restore on `beforeEach` hook? If we already restore on `afterEach` there shouldn't be any new stubs created between the `afterEach` and the `beforeEach` that need to be restored.

Reply:
totally agree